### PR TITLE
Use default VC toolset

### PR DESCRIPTION
**Pull request by dougbu**
- aspnet/AspNetCore-Internal#2246

### CODE REVIEW

**Review by javiercn — APPROVED** [review]
Based on what the VS error tells me and what @jkotalik suggested, this should be ok.

**Comment by jkotalik** [comment]
@javiercn can you try that locally to make sure it works? We won't get validation on CI with this change.

**Comment by jkotalik** [comment]
From @javiercn 
> It's like 6:30pm in Spain

Aka I'll test it shortly.

**Comment by javiercn** [comment]
@jkotalik You're such a :trollface:

I'll test it once its merged
😄

**Comment by jkotalik** [comment]
This the only thing I got working locally. I also think it's the only way this will work on CI.

**Review by jkotalik — APPROVED** [review]

File: src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj [hunk 1]
```diff
@@ -33,26 +33,30 @@
     <UseDebugLibraries>true</UseDebugLibraries>
     <PlatformToolset>v142</PlatformToolset>
     <CharacterSet>Unicode</CharacterSet>
+    <VCToolsVersion />
```
Comment (dougbu) [inline src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj]: Lost of repetition here. Can these be in the "Globals" property group or an unconditional "Configuration" property group?

**Review by dougbu — COMMENTED** [review]
If it works, :ok: but what about Installers\Windows\AspNetCoreModule-Setup/*/*.vcxproj ?

**Comment by jkotalik** [comment]
@dougbu this is what was generated when using the Property Pages in VC++ tooling. I agree it is horrible and I can simplify it, but it is auto generated.

**Comment by jkotalik** [comment]
🆙 📅

**Comment by jkotalik** [comment]
This is blocking people so if this passes, I'm merging btw.

**Comment by jkotalik** [comment]
gah communication errors <_<

**Pull request merged**